executor/importer/import.go

```diff
@@ -920,15 +920,15 @@ func (e *LoadDataController) InitDataFiles(ctx context.Context) error {
 			return exeerrors.ErrLoadDataCantRead.GenWithStackByArgs(GetMsgFromBRError(err2), "failed to read file size by seek")
 		}
 		compressTp := mydump.ParseCompressionOnFileExtension(fileNameKey)
-		dataFiles = append(dataFiles, &mydump.SourceFileMeta{
+		fileMeta := mydump.SourceFileMeta{
 			Path:        fileNameKey,
 			FileSize:    size,
 			Compression: compressTp,
 			Type:        sourceType,
-			// todo: if we support compression for physical mode, should set it to size * compressRatio to better split
-			// engines
-			RealSize: size,
-		})
+			RealSize:    size,
```

Comment:
maybe move the `size` as an argument of `getFileRealSize` so we only need to write to `fileMeta.RealSize` once